timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -47,8 +73,44 @@ services:
           namespace: string
           request: ConjureStartTransactionsRequest
         returns: ConjureStartTransactionsResponse
+      startTransactionsWithWatches:
+        http: POST /stww/{namespace}
+        args:
+          namespace: string
+          request: StartTransactionsWithWatchesRequest
+        returns: StartTransactionsWithWatchesResponse
+        docs: |
+          The returned lock watch state the state taken atomically along with issuing the start timestamps. A failed
+          update means the lock watch state changed too much while start timestamps were being issued, which is
+          indicative of poorly registered lock watches.
       leaderTime:
         http: POST /lt/{namespace}
         args:
           namespace: string
-        returns: LeaderTime
\ No newline at end of file
+        returns: LeaderTime
+      getCommitTimestampWithWatches:
+        http: POST /gctww/{namespace}
```

Comment:
Not sure how much this came out in the RFC/discussions, but I think we might need some form of autobatching around this operation. Otherwise this will cause the load on TimeLock to increase considerably, since currently `getFreshTimestamps` used in committing is autobatched.

Reply:
Makes sense, will make this endpoint batched and add autobatching to get commit timestamp